### DISCUSSION TITLE
ıbax acaba avax gibi olacak?

### DISCUSSION THREAD

**Discussion by Tunc37** [IBAX-io/go-ibax#84] [Ideas]
IBAX projesi AVAX ı geçer mi?

**Comment by hemresenol** [comment]
Şuan ki piyasada biraz zor ama uzun vadede avax gibi olabilir.

**Comment by yaman8121** [comment]
beklemekte fayda var gibi

**Reply by mlkmutlu** [reply]
bence olur

**Comment by cartvecurt** [comment]
Avax örneğinden dolayı buraya çok daha fazla kişi katılıyor, o yüzden yine iyi miktarda kazansak bile ona ulaşacağını sanmıyorum
